packages/extension-webgazer/rollup.config.mjs

```diff
@@ -1,3 +1,3 @@
 import { makeRollupConfigForPlugin } from "@jspsych/config/rollup.mjs";
 
-export default makeRollupConfigForPlugin("jsPsychExtensionWebgazer");
+export default makeRollupConfigForPlugin("jsPsychWebgazerExtension");
```

Comment:
I liked `jsPsychExtensionWebgazer` very much for consistency in the global scope :smiling_face_with_tear: I think it's nice to see whether something is an extension already before you read to the end of it; I'd personally rate consistency of global var names higher than consistency between global vars and class names. What do you think?

Reply:
That's fine with me. I switched it because the plugins don't have a corresponding identifier, but since plugins are much more common I'm OK with having the identifier only on the extensions.